src/NuGet.Indexing/Handlers/LatestListedHandler.cs

```diff
@@ -105,5 +111,11 @@ private static bool GetListed(Document document)
             string listed = document.Get("Listed");
             return (listed == null) ? false : listed.Equals("true", StringComparison.InvariantCultureIgnoreCase);
         }
+
+        private static bool GetSemver2(Document document)
+        {
+            string semverLevel = document.Get("SemverLevel");
```

Comment:
Maybe a IsSemVer2 instead of GetSemVer2? I would expect as  GetSemVer2 to return the version.